[docs-only] Deprecate some web envvars (#6229)
owncloud/ocis: services/web/README.md

If you want to use your custom compiled web client assets instead of the embedded ones, then you can do that by setting the `WEB_ASSET_PATH` variable to point to your compiled files. See [ownCloud Web / Getting Started](https://owncloud.dev/clients/web/getting-started/) and [ownCloud Web / Setup with oCIS](https://owncloud.dev/clients/web/backend-ocis/) for more details.

> ## WebUI Configuration

Reviewer: ```suggestion
## Web UI Configuration
```

> Note that single configuration settings of the embedded WebUI can be defined via `WEB_OPTION_xxx` environment variables. If a json based configuration file is used via the `WEB_UI_CONFIG_FILE` environment variable, these configurations take precedence over single options set.

Reviewer: ```suggestion
Note that single configuration settings of the embedded web UI can be defined via `WEB_OPTION_xxx` environment variables. If a json based configuration file is used via the `WEB_UI_CONFIG_FILE` environment variable, these configurations take precedence over single options set.
```

> ### WebUI Options

Reviewer: ```suggestion
### Web UI Options
```

> Beside theming, the behaviour of the WebUI can be configured via options. See the environment variables `WEB_OPTION_xxx` for more details.

Reviewer: ```suggestion
Beside theming, the behavior of the web UI can be configured via options. See the environment variables `WEB_OPTION_xxx` for more details.
```

> ### WebUI Config File

Reviewer: ```suggestion
### Web UI Config File
```

> When defined via the `WEB_UI_CONFIG_FILE` environment variable, the configuration of the WebUI can be made with a [json based](https://github.com/owncloud/web/tree/master/config) file.

Reviewer: ```suggestion
When defined via the `WEB_UI_CONFIG_FILE` environment variable, the configuration of the web UI can be made with a [json based](https://github.com/owncloud/web/tree/master/config) file.
```
